src/targets/gpu/kernels/include/migraphx/kernels/array.hpp

```diff
@@ -218,14 +218,14 @@ struct array
     {
         for(index_int i = 0; i < N; i++)
         {
-            if(not(x[i] < y[i]))
+            if((x[i] >= y[i]))
```

Comment:
I think its better to write this in terms of `<`.

Reply:
I've reverted this file.